src/icons/collections/IconCollectionsFilled.js

```diff
@@ -8,23 +9,35 @@ type Props = {
     className?: string,
     color?: string,
     height?: number,
+    selected?: boolean,
     /** A text-only string describing the icon if it's not purely decorative for accessibility */
     title?: string | React.Element<any>,
     width?: number,
 };
 
-const IconCollectionsFilled = ({ className = '', color = lightestCharcoal, height = 32, title, width = 32 }: Props) => (
+const IconCollectionsFilled = ({
+    className = '',
+    color = lightestCharcoal,
```

Comment:
can you change this to match one of the BDL colors (closest possible)

Reply:
Sure, will fix